### DISCUSSION TITLE
More Stealth

### DISCUSSION THREAD

**Discussion by DodgeMyRat** [shivaya-dav/DogeRat#46] [General]
How about using a more stealthier app more similar to system apps like used in other RATs ("MainActivity", "Google Play Service")?
Most victims would get suspicious when they see a new app called "YOUTUBE PREMIUM".

However, changing the app name and icon can be done individually with apk editor. Would prefer an "empty" app tho.

**Comment by GILANGDIRGANA** [comment]
I agree with that
